src/phoenix/experimental/evals/functions/binary.py

```diff
@@ -58,4 +51,80 @@ def llm_eval_binary(
         )
 
     responses = model.generate(prompts.to_list(), system_instruction)
-    return responses
+    rail_classes = set(rails)
+    return [
+        (rail_class if (rail_class := resp.strip()) in rail_classes else None) for resp in responses
+    ]
+
+
+def run_relevance_eval(
```

Comment:
the function signature here is kinda different than above (df vs dtataframe) - let's keep it consistent.

Also naming wise shouldn't it be `llm_eval_relevance`?

Reply:
addressed in [d6bdce6](https://github.com/Arize-ai/phoenix/pull/1250/commits/d6bdce60824017f0b1f284064ee99a35f74d0dc5)

Reply:
I'm not clear yet that these functions are actually of the same kind/ should be named similarly. They're serving different purposes at the moment. One is internal/ used for experiment and tuning prompts. The other is used on OpenInference tracing data.